.github/workflows/breaking-change-description-title-update.yml

```diff
@@ -9,7 +9,7 @@ on:
 
 # cancel redundant builds
 concurrency:
-  group: "update-breaking-change-${{ github.head_ref }}"
+  group: "update-breaking-change-${{ github.ref_name }}"
```

Comment:
instead of ref_name can this be just `ref`

I am probably being overly cautious here but if there are two prs with the same name but one from a fork and one from this repo, would concurrency work with ref_name or do we need to know origin/fork information

Reply:
Sure, can update